### ISSUE TITLE
Adjusting learning rate

### ISSUE THREAD

**Issue by Junonia** [pytorch/examples#64]
I was not able to get the adjust learning rate working unless I change the code at line 266 in main.py from

`    for param_group in optimizer.state_dict()['param_groups']:
`

to 

`    for param_group in optimizer.param_groups:
`

**Comment by apaszke** [comment]
Ah, you're right! It got broken by a recent change! Thanks for the report! Please send a PR, if you have a moment, otherwise I can do it later today.

**Comment by soumith** [comment]
this is fixed in master
